ui/src/components/LoginForm/LoginForm.css

```diff
@@ -1,18 +1,17 @@
-form .title {
+.form-title {
     font-weight: bold;
+    color: var(--dark-blue);
     margin-top: 20px;
     margin-bottom: 10px;
 }
 
-form label {
-    display: block;
-    margin-bottom: 5px;
+.login-card {
+    margin: 10px;
+    margin-left: auto;
+    margin-right: auto;
+    max-width: 700px;
 }
 
-form span {
-    margin-right: 10px;
+.login-button {
+    margin-top: 10px !important;
```

Comment:
nit: !important is a sign that we have too many CSS that are overriding one another. No action here but definitely watch out for them